projects/backend/controllers/issue.ts

```diff
@@ -55,8 +55,12 @@ export const issueController = (req: Request, res: Response) => {
         .catch(e => console.error(e))
 }
 
-const getIssuesSummary = async (): Promise<Attempt<IssueSummary[]>> => {
-    const issueKeys = await s3List('daily-edition/')
+export const getIssuesSummary = async (
+    pageSize = 7,
+    /* mock for tests */
+    getIssueKeys = () => s3List('daily-edition/'),
```

Comment:
Can we do this without polluting everything with dependency injection?

Reply:
I think we have different opinions on this, I prefer to see the dependencies of a function as arguments and also have the benefit of type safety in test. However, if, as a team, we prefer mocking, then I'm happy to do that.

Reply:
I'm with @RichieAHB although only because I would follow a similar pattern in Scala.